pkg/cli/server_test.go

```diff
@@ -36,7 +32,7 @@ const (
 func TestServerCommand(t *testing.T) {
 	t.Parallel()
 
-	testPrivateKeyString, _ := testGeneratePrivateKey(t)
+	testRSAPrivateKeyString, _ := keyutil.TestGenerateRsaPrivateKey(t)
```

Comment:
If RSA key generation is slow (not sure about this), maybe it makes sense to just hardcode a test key rather than regenerate it for each test?

Reply:
Yeah, I can hard code a pem in a file and tests can read from it. Or just make it a const in `keyutil`. But I think the key generation is not slow, and not too many cases are using this.

Reply:
Yeah, you're right, it's not too slow. I just benchmarked it at 115ms.